scenes/music_puzzle/music_puzzle.gd

```diff
@@ -68,7 +68,10 @@ func _on_note_played(note: String) -> void:
 
 func _get_configuration_warnings() -> PackedStringArray:
 	if melodies.size() != fires.size():
-		var fmt = "There should be one fire for each melody, but there are %d melodies and %d fires"
-		return [fmt % [melodies.size(), fires.size()]]
+		var fmt: String = """
+			There should be one fire for each melody, \
+			but currently there are %d melodies and %d fires.
+		"""
+		return [fmt.strip_edges() % [melodies.size(), fires.size()]]
```

Comment:
Adding the `: String` pushed this over gdlint's 100-column limit. The kind of eccentric use of a multiline string plus `strip_edges()` is to avoid some of gdformat's grosser formatting choices. But I wonder if I can find something neater…

Reply:
I think this is the best I can do.